### PR TITLE
adding monitoring scripts for slave

### PR DESCRIPTION
**Pull request by rshade**

### CODE REVIEW

**Comment by rshade** [comment]
closes #43

File: templates/default/exec.conf.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+LoadPlugin exec
+<Plugin exec>
+  # userid plugin executable plugin args
+  #   Exec "my_user" "/usr/lib/collectd/plugins/mycpuload.rb"
+  <% node['kabam-rs-mysql']['exec'].each do |exec| %>
```
Comment (cdwilhelm) [inline templates/default/exec.conf.erb]: this should not be for kabam

File: attributes/default.rb [hunk 2]
```diff
@@ -66,3 +66,8 @@
 
 # Dump file location in repository to import.
 default['rs-mysql']['import']['dump_file'] = nil
+
+# Sets up empty collectd options for v5
+default['rs-mysql']['collectd']['mysql']['User'] = 'root'
```
Comment (cdwilhelm) [inline attributes/default.rb]: should the attributes be lower case

File: recipes/slave.rb [hunk 3]
```diff
@@ -136,3 +136,8 @@ class Chef::Recipe
   backup false
   action :delete
 end
+
+node.default['rs-mysql']['collectd']['mysql']['SlaveStats'] = 'true'
```
Comment (cdwilhelm) [inline recipes/slave.rb]: attributes lower case?

File: recipes/slave_monitors.rb [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+
+template '/usr/local/bin/mysql_seconds_behind_master.rb' do
+  source 'mysql_seconds_behind_master.rb.erb'
+  owner 'root'
+  group 'root'
+  mode 0755
+  action :create
+end
+
+node.set['kabam-rs-mysql']['exec'] = ['Exec "mysql_monitor" "/opt/chef/embedded/bin/ruby" "/usr/local/bin/mysql_seconds_behind_master.rb"']
```
Comment (cdwilhelm) [inline recipes/slave_monitors.rb]: kabam?

**Pull request merged**